morse.rb

```diff
@@ -0,0 +1,84 @@
+MORSE_CODE = {
+  '.-' => 'a',
+  '-...' => 'b',
+  '-.-.' => 'c',
+  '-..' => 'd',
+  '.' => 'e',
+  '..-.' => 'f',
+  '--.' => 'g',
+  '....' => 'h',
+  '..' => 'i',
+  '.---' => 'j',
+  '-.-' => 'k',
+  '.-..' => 'l',
+  '--' => 'm',
+  '-.' => 'n',
+  '---' => 'o',
+  '.--.' => 'p',
+  '--.-' => 'q',
+  '.-.' => 'r',
+  '...' => 's',
+  '-' => 't',
+  '..-' => 'u',
+  '...-' => 'v',
+  '.--' => 'w',
+  '-..-' => 'x',
+  '-.--' => 'y',
+  '--..' => 'z',
+  ' ' => ' ',
+  '.----' => '1',
+  '..---' => '2',
+  '...--' => '3',
+  '....-' => '4',
+  '.....' => '5',
+  '-....' => '6',
+  '--...' => '7',
+  '---..' => '8',
+  '----.' => '9',
+  '-----' => '0'
+}.freeze
+
+# Create a method to decode a Morse code character,
+# takes a string parameter, and return the corresponding
+# character in uppercase.
+
+def decode_char(char)
+  MORSE_CODE[char].capitalize
+end
+
+# Create a method to decode an entire word in Morse code, takes a string parameter,
+# and return the string representation.
+# Every character in a word will be separated by a single space.
+
+def decode_word(word)
+  result = ''
+  word.split.each do |morse_char|
+    decoded_char = decode_char(morse_char)
+    result += decoded_char.to_s
+  end
+  result
+end
+
+# Create a method to decode the entire message in Morse code, takes a string parameter,
+# and return the string representation. Every word will be separated by 3 spaces
+
+def split_phrase(phrase)
+  phrase.split
+end
+
+def decode(phrase)

```

Comment:
- [ ] Your program is not decoding the sentences as intended. Spaces between sentences are not showing, and also spaces seem to appear between each character of a word. Kindly compare the attached screenshots and improve upon this section accordingly.

**Your output:**
![image](https://user-images.githubusercontent.com/12745474/181477408-1eaf0b69-f40b-4d07-b11c-cd413fc9a24b.png)


**Expected output:**
![image](https://user-images.githubusercontent.com/12745474/181477604-152bfe3d-efc4-461a-ad16-aea4388e7367.png)